alwaysontop/main.js

```diff
@@ -38,27 +56,55 @@ function onAlwaysOnTopWindow(
         const win = event.newGuest = new BrowserWindow(
             Object.assign(options, {
                 backgroundColor: 'transparent',
-                width: SIZE.width,
-                height: SIZE.height,
                 minWidth: SIZE.width,
                 minHeight: SIZE.height,
-                maxWidth: SIZE.width,
-                maxHeight: SIZE.height,
                 minimizable: false,
                 maximizable: false,
-                resizable: false,
+                resizable: true,
                 alwaysOnTop: true,
                 fullscreen: false,
                 fullscreenable: false,
                 skipTaskbar: true,
                 titleBarStyle: undefined,
                 frame: false,
                 show: false
-            }, getPosition())
+            }, getPosition(), getSize())
         );
         win.once('ready-to-show', () => {
             win.showInactive();
         });
+
+        //for macOS we use the built-in setAspectRatio on resize, for other we use custom implementation
+        if (os.type() === 'Darwin') {
+            win.setAspectRatio(16/9);
+        } else {
+            win.on('will-resize', (e, newBounds) => {
+                oldSize = win.getSize();
+                const mousePos = robot.getMousePos();
```

Comment:
Do we really need to use robotjs for this? Isn't it https://github.com/electron/electron/blob/master/docs/api/screen.md#screengetcursorscreenpoint do the job?

Reply:
Please see following image
First is robotjs detection, second is electron screen detection on a monitor having scale 125%. 
Please note that robotjs correctly shows the y axe as 0 in this case, and electron screen detects it wrongly as 200 something. I think this wrong detection electron-side causes all the bugs reported on Electron related to scaled monitors on Windows.

![image](https://user-images.githubusercontent.com/39557534/61306846-e9a15000-a7f5-11e9-8bff-4a2d059b019c.png)